modules/nw-configuring-sriov-rdma-cni.adoc

```diff
@@ -0,0 +1,177 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-sriov-rdma-cni.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-sriov-configuring-sriov-rdma-cni_{context}"]
+= Configuring SR-IOV RDMA CNI
+
+You can configure an RDMA CNI on SR-IOV.
```

Comment:
can we have an intro here with information about this feature and when the user needs to use it?